packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/AdvancedTxDetails.tsx

```diff
@@ -66,7 +66,7 @@ export const AdvancedTxDetails = ({
                 <Tabs.Item id="amount" onClick={() => setSelectedTab('amount')}>
                     <Translation id="TR_TX_TAB_AMOUNT" />
                 </Tabs.Item>
-                {network.networkType !== 'ripple' && (
+                {(network.networkType !== 'ripple') && (
```

Comment:
no need to change

Reply:
This issue was addressed during the rebase.